### ISSUE TITLE
Fix holographic for gamelauncher 

### ISSUE THREAD

**Issue by kRHYME7** [HyDE-Project/HyDE#12]
Dynamically follow the monitor aspect ratio for the holograph.

**Comment by Anshul-007** [comment]
Also the mode is set to 1 as default and there is no UI to change it by user right?

**Comment by imtheabdi** [comment]
> Also the MODE is set to 1 as default and there is no UI to change it by user right?

i keep forgetting that this script exists.

**Comment by kRHYME7** [comment]
you wanted a rofi selector for this too?

**Comment by Keegan-Wright** [comment]
Ah yeah mine are 2560x1440 haha sadly not wide enough!

I'll have a poke around through hyprdots and see if I can get something worked out :) I'll bundle it with a game Launcher selector too so the other themes can be used easily

**Comment by kRHYME7** [comment]
> 2560x1440

I see you got the same aspect ratio with hyprdots therefore it works ootb. Can you try the latest dev changes for the game launcher?

So what changed is I modify all px units to scale using em or %.

**Comment by Keegan-Wright** [comment]
Below is the master branch
![Image](https://github.com/user-attachments/assets/f36547fa-361c-42f4-99ff-a088d7b80a07)

And this one is running directly from dev
![Image](https://github.com/user-attachments/assets/984259c5-16fb-4bff-9da2-c8dbc94736e2)

**Comment by kRHYME7** [comment]
yup that would be helpful. Though vertical monitors need a separate .rasi file and different set of config.

I tried it with 3 monitors and it looks scaled for me.

**Comment by Keegan-Wright** [comment]
I'll try on all the monitors I have at home and vertical too once I get a chance, might be tomorrow

**Comment by Keegan-Wright** [comment]
Okay it looks all well and good on all monitors and resolutions i've tried. had a poke around with a vertical monitor too

![Image](https://github.com/user-attachments/assets/71c47376-58a0-4197-b2ea-e737b4b35f14)
Little bit rough but i assume that's the rasi file?

**Comment by kRHYME7** [comment]
>  Little bit rough but i assume that's the rasi file? 

yeah. And unlike CSS it is not aware of the element orientation. we can just script it generating tons of varibles or just create a separate vertical monitor rasi file that can scale.
